ruby/computer_science/a_very_brief_intro_to_cs.md

```diff
@@ -22,7 +22,7 @@ This section contains a general overview of topics that you will learn in this l
 <div class="lesson-content__panel" markdown="1">
   1. Watch [Introduction to Algorithms by David Malan](https://www.youtube.com/watch?v=6hfOvs8pY1k) on TedEd to see how to think about algorithms.
   2. Watch [What is an Algorithm?](https://youtu.be/e_WfC8HwVB8) on YouTube for a more structured look at solving problems using algorithms.
-  3. Read [this Quora answer about the importance of algorithms in web development](https://qr.ae/py3NAc) to get some context for why we're going over this stuff.
+  3. Read this Quora answer about [the importance of algorithms in web development](https://qr.ae/py3NAc) to get some context for why we’re going over this stuff.
```

Comment:
```suggestion
  3. Read this Quora answer about [the importance of algorithms in web development](https://qr.ae/py3NAc) to get some context for why we're going over this stuff.
```
Just a small suggestion here to keep the apostrophe character as `'` (which we use throughout the curriculum) rather than `’` (which I guess is how it's formatted in Google Docs maybe? And that made it over in copy/paste)